examples/tests_ci/run_test_sink_synchro.py

```
@@ -206,5 +206,41 @@ def main():
+
+    mismatch = False
+    for i in range(len(dat)):
+        if not np.isclose(dat[i], ref_sums[i], rtol=1e-14, atol=1e-18):
+            abs_diff = np.abs(dat[i] - ref_sums[i])
+            rel_diff = abs_diff / np.abs(ref_sums[i])
+            print(f"sum[{i}] mismatch: got {dat[i]}, expected {ref_sums[i]}")
+            print(f"  max abs diff={np.max(abs_diff)}")
+            print(f"  max rel diff={np.max(rel_diff)}")
+            mismatch = True
+    if mismatch:
+        raise RuntimeError("sums mismatch")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)